<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Track_Options</class>
 <widget class="QWidget" name="Track_Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="key_pixmap">
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="end_pixmap">
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QSlider" name="size_slider">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="key_slider">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSlider" name="end_slider">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QComboBox" name="tones_cb">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>115</width>
       <height>64</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>C</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/0</normaloff>:/tones/0</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/1</normaloff>:/tones/1</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/2</normaloff>:/tones/2</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/3</normaloff>:/tones/3</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bb</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/4</normaloff>:/tones/4</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/5</normaloff>:/tones/5</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Eb</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/6</normaloff>:/tones/6</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/7</normaloff>:/tones/7</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ab</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/8</normaloff>:/tones/8</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/9</normaloff>:/tones/9</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Db</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/10</normaloff>:/tones/10</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F#</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/11</normaloff>:/tones/11</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gb</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/12</normaloff>:/tones/12</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C#</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/13</normaloff>:/tones/13</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cb</string>
      </property>
      <property name="icon">
       <iconset resource="../icons.qrc">
        <normaloff>:/tones/14</normaloff>:/tones/14</iconset>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
